### PR TITLE
Fix up schema editor for AE.

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/automate/test_domain_priority.py -v --long-running }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit bf04a58681c65e4027b7f236dfa6d2718b23f82c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/automate/explorer.py [hunk 1]
```diff
@@ -527,22 +527,31 @@ def get_form(self, blank=False):
...
                 else:
                     plural = "s"
-                return fmt % (plural, row_id)
+                return fmt % (plural, row_id if underscore else row_id.lstrip("_"))
 
             def remove(loc):
                 """Return a callable that clicks but still allows popup dismissal"""
                 return lambda _: sel.click(loc, wait_ajax=False)
 
+            if version.current_version() > "5.5":
```
Comment (psav) [inline cfme/automate/explorer.py]: why no version.pick love?

File: cfme/automate/explorer.py [hunk 2]
```diff
@@ -527,22 +527,31 @@ def get_form(self, blank=False):
...
+                datatype = AngularSelect(loc("field%s_datatype%s", underscore=False))
+            else:
+                datatype = DHTMLSelect(loc("//div[@id='field%s_datatype_id%s']"))
+
             return Form(
                 fields=[('name_text', Input(loc('field%s_name%s'))),
-                        ('type_select', DHTMLSelect(loc("//div[@id='field%s_aetype_id%s']"))),
-                        ('data_type_select',
-                         DHTMLSelect(loc("//div[@id='field%s_datatype_id%s']"))),
+                        ('type_select', aetype),
```
Comment (psav) [inline cfme/automate/explorer.py]: Since we are returning a form here, can't we just leave verpick dicts?

Comment (mfalesni) [inline cfme/automate/explorer.py]: Too much nesting?

**Comment by dajoRH** [comment]
Lint report for commit 8429fbff65755ac2ac5c51f3455c9327496d8f25:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eb1a4f36c90fc95fe1de1d0c667bd9890fe5c5a2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 886e0d5e785d6552b374e6564c5d5a85fc1d9e82:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**